packages/labnative/package.json

```diff
@@ -28,6 +29,9 @@
     "metro-react-native-babel-preset": "^0.58.0",
     "react-test-renderer": "16.13.1"
   },
+  "peerDependencies": {
+    "@react-native-picker/picker": "^1.16.6"
```

Comment:
It should either be a `peerDependency` OR a `dependency`, you can find more about it [here](https://flaviocopes.com/npm-peer-dependencies/#:~:text=peerDependencies%20are%20different.,include%20it%20as%20its%20dependency.).

Also, this is not the package that we publish, this should be added under the `@gympass/yoga` respective `package.json` (this is the `labnative` one).

Reply:
Thanks for pointing out. I sent a re-request :)